db/src/main/java/net/bible/android/database/AppDatabase.kt

```diff
@@ -26,7 +26,7 @@ import net.bible.android.database.readingplan.ReadingPlanEntities
 
 import java.util.*
 
-const val DATABASE_VERSION = 32
```

Comment:
Version incremented but no migration?

Reply:
The migration is already there. I can find later which commit it should have been incremented, but I think this was missed.

Reply:
It looks like it has been forgotten indeed. Then it's okay to fix that.